docs/windows.md

```diff
@@ -0,0 +1,29 @@
+# Windows Installation
```

Comment:
Can you link to this in `docs/getting-started.md`, add it to the table of contents in `docs/README.md`, and add another reference for it in `production/README.md`?